src/components/button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Button = ({
+  children,
+  secondary,
+  loading,
+  disabled,
+  accessibilityLabel,
+  onClick = () => { },
+  to,
+  className
+}) => {
+  const buttonClassNames = classNames(
+    'button',
```

Comment:
A good pattern that I think we used for Pitchy is prefixing the type of element it is.

`c-` for component
`u-` for utility

So could become `c-btn` ?